src/framework/mpe/tests/multinotearticulationstest.cpp

```diff
@@ -281,7 +281,8 @@ TEST_F(MultiNoteArticulationsTest, CrescendoPattern)
 
     for (size_t i = 0; i < noteCount; ++i) {
         // [GIVEN] Crescendo articulation applied on the note
-        ArticulationAppliedData crescendoApplied(crescendoMeta, 25 * ONE_PERCENT * i, 25 * ONE_PERCENT * (i + 1));
+        ArticulationAppliedData crescendoApplied(crescendoMeta, 25 * ONE_PERCENT * static_cast<int>(i),
+                                                 25 * ONE_PERCENT * (static_cast<int>(i) + 1));
```

Comment:
here too?

Reply:
you could just do the cast for the noteCount variable, I think it'll be enough here

Reply:
You mean `for (int i = 0; i < static_cast<int>(noteCount); ++i) {`?
That's not what we do further up, so not good for consistency

Reply:
Done. That got rid of a couple more casts

Reply:
`Or int noteCount = static_cast<int>(m_initialData.size());`

yes, this one

Reply:
Done. That got rid of a couple more casts